src/CONST.js

```diff
@@ -1122,6 +1122,10 @@ const CONST = {
             // eslint-disable-next-line no-misleading-character-class
             /[\n\s,/?"{}[\]()&^%$#<>!*\p{Extended_Pictographic}\u200d\u{1f1e6}-\u{1f1ff}\u{1f3fb}-\u{1f3ff}\u{e0020}-\u{e007f}\u20E3\uFE0F]|[#*0-9]\uFE0F?\u20E3/gu,
 
+        SPACE_OR_EMOJI:
+            // eslint-disable-next-line no-misleading-character-class
+            /(\s+|(?:[\p{Extended_Pictographic}\u200d\u{1f1e6}-\u{1f1ff}\u{1f3fb}-\u{1f3ff}\u{e0020}-\u{e007f}\u20E3\uFE0F]|[#*0-9]\uFE0F?\u20E3)+)/gu,
```

Comment:
```
\p{Extended_Pictographic}\u200d\u{1f1e6}-\u{1f1ff}\u{1f3fb}-\u{1f3ff}\u{e0020}-\u{e007f}\u20E3\uFE0F]|[#*0-9]\uFE0F?\u20E3
```

is repeated 4 times in this file, would it make sense to somehow extract it in its own constant? 

Or said another way, can we reuse the `EMOJIS` constant here instead of copy pasting it?

Reply:
@flodnv 
Yes. In order to extract it, we have to extract it as string and use `new RegExp` syntax, but I found that there is no such expression though it has been repeated 3 times already. So I guessed it's intentional.

Do you still want to extract EMOJIS constant as string?

Reply:
> I guessed it's intentional.

I guessed it was not intentional 😄 

What do you think @s77rt ?

Reply:
I doubt this is intentional. Can we extract `\p{Extended_Pictographic}\u200d\u{1f1e6}-\u{1f1ff}\u{1f3fb}-\u{1f3ff}\u{e0020}-\u{e007f}\u20E3\uFE0F]|[#*0-9]\uFE0F?\u20E3`? If yes then let's do it

Reply:
@flodnv @s77rt 

For now, `unicode property escape` cannot be used in `new RegExp` constructor in our project. That's why those were not refactored yet

Reply:
Got it. Sorry for the noob question -- I understand that we cannot use it in `new RegExp`, but why does this matter here? 

Asked another way, why can we not have (pseudo code):
```
EMOJIS: /[\p{Extended_Pictographic}\u200d\u{1f1e6}-\u{1f1ff}\u{1f3fb}-\u{1f3ff}\u{e0020}-\u{e007f}\u20E3\uFE0F]|[#*0-9]\uFE0F?\u20E3/gu,

SPACE_OR_EMOJI: /(\s+|(?:EMOJI)+)/gu,
```

Reply:
@flodnv 
`/.../gu` isn't a string, it's a `RegExp` object. We can't embed it inside another RegExp object like that.

Reply:
Ohhhh 👍 Thanks!